### PR TITLE
Virtual select box fix tangible theme

### PR DESCRIPTION
**Pull request by zaucker**
- Syntax fix for missing "list-search-highlight" element in tangible
- Use background highlighting instead of red/bold (got lost somehow in PR #10077) 
is

### CODE REVIEW

**Comment by zaucker** [comment]
Will fix/finish PR #10080 after this one is merged.

File: framework/source/class/qx/theme/indigo/Appearance.js [hunk 1]
```diff
@@ -183,8 +183,8 @@ qx.Theme.define("qx.theme.indigo.Appearance",
     "list-search-highlight" : {
       style : function(states) {
         return {
-          color      : '#FF0000',
-          fontWeight : 'bold'
+          text-decoration-style: dotted,
```
Comment (cboulanger) [inline framework/source/class/qx/theme/indigo/Appearance.js:186]: Does this work, as javascript? Don't you have to put it into quotation marks?

Comment (cboulanger) [reply framework/source/class/qx/theme/indigo/Appearance.js:186]: I mean: wouldn't `text-decoration-style` be interpreted as a subtraction of unknown variables?

File: framework/source/class/qx/theme/classic/Appearance.js [hunk 2]
```diff
@@ -2714,8 +2714,8 @@ qx.Theme.define("qx.theme.classic.Appearance",
     "list-search-highlight" : {
       style : function(states) {
         return {
-          color      : '#FF0000',
-          fontWeight : 'bold'
+          text-decoration-style: dotted,
+          text-decoration-line: underline
```
Comment (oetiker) [inline framework/source/class/qx/theme/classic/Appearance.js:2718]:
```suggestion
          textDecorationStyle: 'dotted',
          textDecorationLine: 'underline'
```

File: framework/source/class/qx/theme/indigo/Appearance.js (hunk 1 above)
Comment (oetiker) [inline framework/source/class/qx/theme/indigo/Appearance.js:186]:
```suggestion
          textDecorationStyle: 'dotted',
```

File: framework/source/class/qx/theme/modern/Appearance.js [hunk 3]
```diff
@@ -3054,8 +3054,8 @@ qx.Theme.define("qx.theme.modern.Appearance",
     "list-search-highlight" : {
       style : function(states) {
         return {
-          color      : '#FF0000',
-          fontWeight : 'bold'
+          text-decoration-style: dotted,
+          text-decoration-line: underline
```
Comment (oetiker) [inline framework/source/class/qx/theme/modern/Appearance.js:3058]:
```suggestion
          textDecorationStyle: 'dotted',
          textDecorationLine: 'underline'
```

File: framework/source/class/qx/theme/simple/Appearance.js [hunk 4]
```diff
@@ -2794,8 +2794,8 @@ qx.Theme.define("qx.theme.simple.Appearance",
     "list-search-highlight" : {
       style : function(states) {
         return {
-          color      : '#FF0000',
-          fontWeight : 'bold'
+          text-decoration-style: dotted,
+          text-decoration-line: underline
```
Comment (oetiker) [inline framework/source/class/qx/theme/simple/Appearance.js:2798]:
```suggestion
          textDecorationStyle: 'dotted',
          textDecorationLine: 'underline'
```

File: framework/source/class/qx/theme/tangible/Appearance.js [hunk 5]
```diff
@@ -2649,21 +2649,21 @@ qx.Theme.define("qx.theme.tangible.Appearance", {
...
-    style : function(states) {
-      return {
-        color      : '#FF0000',
-        fontWeight : 'bold'
-      };
+    "list-search-highlight" : {
+      style : function(states) {
+        return {
+          text-decoration-style: dotted,
+          text-decoration-line: underline
```
Comment (oetiker) [inline framework/source/class/qx/theme/tangible/Appearance.js:2664]:
```suggestion
          textDecorationStyle: 'dotted',
          textDecorationLine: 'underline'
```

**Review by oetiker — CHANGES_REQUESTED** [review]

File: framework/source/class/qx/theme/indigo/Appearance.js (hunk 1 above)
Comment (zaucker) [reply framework/source/class/qx/theme/indigo/Appearance.js:186]: Of course not ... fixed.

**Comment by zaucker** [comment]
Fixed syntax ...

**Review by cboulanger — APPROVED** [review]

**Review by oetiker — APPROVED** [review]

**Review by johnspackman — APPROVED** [review]

**Review by hkollmann — APPROVED** [review]

**Review by hkollmann — APPROVED** [review]

**Review by level420 — APPROVED** [review]

**Comment by level420** [comment]
thank you @zaucker

**Pull request merged**